deploy/olm-catalog/akka-cluster-operator/0.2.0/akka-cluster-operator.v0.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,300 @@
+              verbs:
+                - update
+            - apiGroups:
+                - app.lightbend.com
+              resources:
+                - "*"
+              verbs:
+                - "*"
+          serviceAccountName: akka-cluster-operator
+      deployments:
+        - name: akkaclusteroperator-deployment
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: akka-cluster-operator
+            template:
+              metadata:
+                labels:
+                  name: akka-cluster-operator
+              spec:
+                containers:
+                  - command:
+                      - akka-cluster-operator
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            apiVersion: v1
+                            fieldPath: 'metadata.annotations[''olm.targetNamespaces'']'
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            apiVersion: v1
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: akka-cluster-operator
+                    image: >-
+                      lightbend-docker-registry.bintray.io/lightbend/akkacluster-operator:latest
+                    imagePullPolicy: Always
+                    name: akka-cluster-operator
+                    resources:
+                      limits:
+                        cpu: 200m
+                        memory: 100Mi
+                      requests:
+                        cpu: 100m
+                        memory: 50Mi
+                serviceAccount: akka-cluster-operator
+                serviceAccountName: akka-cluster-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - Akka
+    - Akka Cluster
+    - Lightbend
+  labels:
+    Name: AkkaClusterOperator
+  maturity: alpha
+  minKubeVersion: 1.11.0
+  provider:
+    name: 'Lightbend, Inc.'
+  keywords: ["Akka", "Akka Cluster", "Lightbend"]
+  maintainers:
+    - name: 'Lightbend, Inc.'
+      email: info@lightbend.com
+  links:
+    - name: Intro to Akka
+      url: https://doc.akka.io/docs/akka/current/guide/introduction.html
+    - name: Intro to Akka Cluster
+      url: https://doc.akka.io/docs/akka/current/common/cluster.html
+    - name: Akka Cluster demo application
+      url: https://github.com/lightbend/akka-java-cluster-openshift
+    - name: Deploying a Lagom application to OpenShift
+      url: https://developer.lightbend.com/guides/openshift-deployment/lagom/index.html
+  version: 0.2.0
+  replaces: akka-cluster-operator.v0.0.1
```

Comment:
Fwiw, I never had good luck specifying a `replaces` value.  I ended up being hesitant to use it...